rules_building_block/command_and_control_linux_port_knocking_reverse_connection.toml

```diff
@@ -0,0 +1,104 @@
+[metadata]
+bypass_bbr_timing = true
+creation_date = "2023/10/24"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/10/24"
+
+[rule]
+author = ["Elastic"]
+building_block_type = "default"
+description = """
+Monitors for a sequence of network activity on atypical ports, prior to receiving a single packet on such a non-standard
+port, which potentially indicates signal port knocking activity. Port knocking is a covert method of externally opening
+ports by sending a sequence of packets to previously closed ports, which adversaries may leverage to discreetly gain
+access without directly alerting traditional monitoring systems.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Linux Reverse Connection through Port Knocking"
+references = ["https://github.com/f0rb1dd3n/Reptile"]
+risk_score = 21
+rule_id = "86c3157c-a951-4a4f-989b-2f0d0f1f9518"
+severity = "low"
+tags = [
+        "Domain: Endpoint",
+        "OS: Linux",
+        "Use Case: Threat Detection",
+        "Tactic: Command and Control",
+        "Data Source: Elastic Defend",
+        "Rule Type: BBR"
+        ]
+timestamp_override = "event.ingested"
+type = "eql"
+query = '''
+sequence by host.id with maxspan=10s
```

Comment:
the query start with a pretty generic condition (process.name/path any) is there a way to optimize it a bit for performance ?